package.json

```diff
@@ -35,6 +35,7 @@
 		"path"
 	],
 	"dependencies": {
+		"escape-string-regexp": "^1.0.5",
```

Comment:
Why v1? You can do v2 without even increasing the required Node.js version.

Reply:
I don't recall specifying a version. This is what npm installed. I'll bump it to 2.0.